rebar.config

```diff
@@ -2,13 +2,11 @@
 {edoc_opts, [{dir, "edoc"}]}.
 
 {deps,
- [{ranch, "",
-   {git, "https://github.com/ninenines/ranch.git", {tag, "1.1.0"}}}
- ,{cowlib, ".*", {git, "https://github.com/ninenines/cowlib.git", {tag, "0.4.0"}}}
+ [{ranch, "1.6.1"}
```

Comment:
Could this upgrade cause trouble because of [this change](https://github.com/ninenines/ranch/commit/b2b099627424ce42b7f0ac02e5ddd8d0bf2c3381) in ranch?

Reply:
Ah damn right it could. I always forget this total dipshit of a move to switch a blacklist for a whitelist in a minor version. God damn it.

The main problem would be with ranch proxy protocol rather than other stuff, right?

Reply:
Yes, I believe so. `ranch_proxy_ssl` does its own filtering though. Wondering if it's that much of a problem.

Reply:
Probably could move the whitelist into the router.

Reply:
yeah that's what I was wondering. It could be worthwhile to align the newer copies with more modern semantics as well, or make them agnostic to the change for better portability.

Reply:
I just double-checked and I can't find a call within ranch_proxy_ssl nor vegur that seems to use these features directly. I'm curious as to whether the call that once gave trouble was in non-public heroku repos, but for now I think this is safe to merge.

The filtering done in ranch_proxy_ssl already separates SSL opts from non-SSL opts, and therefore plays along with Ranch's own filtering by pre-removing bad entries it wouldn't support.

Maybe otherwise there's some out of date option, but I didn't find it at a glance.